src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -441,17 +479,17 @@ private static void handleCloningFailed(List<RepoConfiguration> configs, RepoLoc
     }
 
     /**
-     * Adds {@code failedConfig} that failed analysis into the list of errors in the summary report and
-     * removes {@code failedConfig} from the list of {@code configs}.
+     * Adds {@code failedConfig} that failed analysis into the list of errors in the summary report along with
+     * an {@code errorMessage} and removes {@code failedConfig} from the list of {@code configs}.
      */
     private static void handleAnalysisFailed(List<RepoConfiguration> configs, RepoConfiguration failedConfig,
             String errorMessage) {
         handleFailedConfigs(configs, Collections.singletonList(failedConfig), errorMessage);
     }
 
     /**
-     * Adds {@code failedConfigs} that failed cloning/analysis into the list of errors in the summary report and
-     * removes {@code failedConfigs} from the list of {@code configs}.
+     * Adds {@code failedConfigs} that failed cloning/analysis into the list of errors in the summary report along
+     * with an {@code errorMessage} and removes {@code failedConfigs} from the list of {@code configs}.
      */
     private static void handleFailedConfigs(
             List<RepoConfiguration> configs, List<RepoConfiguration> failedConfigs, String errorMessage) {
```

Comment:
Should move the `configs` and `failedConfigs` variables to line 494, then keep only the `errorMessage` variable in line 495.